zipline/api.pyi

```diff
@@ -545,6 +545,8 @@ def schedule_function(func, date_rule=None, time_rule=None, half_days=True, cale
         The rule for the times to execute this function.
     half_days : bool, optional
         Should this rule fire on half days?
+    calendar : Sentinel, optional
```

Comment:
FYI We regenerate this file when we [release](http://www.zipline.io/release-process.html#updating-the-python-stub-files) a new version of zipline.